src/lib/guides/StarterKits/getting-started.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Getting Started with Wharfkit Starter Kits
+category: StarterKits
+published: true
+requiresReview: true
+---
+
+# Introduction to Wharfkit Starter Kits
+
+The Starter kits are a great way to start developing applications on Wharfkit. They offer a convenient way to install all the essential Wharfkit packages at once, while also providing `sessionKit` instances that can be used quickly instantiate sessions in your applications.
+
+The available kits are `@wharfkit/starter`, `@wharfkit/starter-eos`, `@wharfkit/starter-telos`, `@wharfkit/starter-wax`, and `@wharfkit/starter-jungle`.
+
+> **NOTE**: This guide is current as of December 2023 and reflects the latest versions of Wharfkit Starter Kits.
+
+## Getting Started
+
+Ensure [Node.js](https://nodejs.org/en/) is installed, as it is a prerequisite for using Wharfkit packages.
+
+### Step 1: Installation
+
+Select and install the Starter kit that best matches your blockchain project's needs:
+
+For a version of the Starter kit that will work with all the main Antelope blockchains:
+```bash
+npm install --save @wharfkit/starter
+# or
+yarn add @wharfkit/starter
+```
+
+For a version of the Starter kit that is tailored to a specific blockchain:
+```bash
+npm install --save @wharfkit/starter-[blockchain]
+# or
+yarn add @wharfkit/starter-[blockchain]
+
+# Replace [blockchain] with 'eos', 'telos', 'wax', or 'jungle'
+```
+
+### Step 2: Setting Up Your Project
+
+After installation, import the `sessionKit` instance and other necessary modules from the necessary Wharf packages:
+
+```js
+import { ContractKit } from '@wharfkit/contract';
+import { AccountKit } from '@wharfkit/account';
+// Import other modules as needed
+```
+
+### Step 3: Using SessionKit
+
+Use the `sessionKit` instance to log the user in and create a Wharf session:
+
+```js
+import { sessionKit } from '@wharfkit/starter';
+
+const session = await sessionKit.login(); // Initiates a login request
+```
```

Comment:
This is lacking a lot of information, but perhaps we could link to one of the other guides to fill in the gaps?

More detail should probably be included here that explains how its already setup with API endpoints and plugins. We can then link off to other sections for more information.

Reply:
Yeah, that makes sense!